back/app/services/events/ics_generator.rb

```diff
@@ -24,7 +26,7 @@ def generate_calendar(events, preferred_locale)
 
     def empty_calendar
       Icalendar::Calendar.new.tap do |cal|
-        cal.prodid = 'CitizenLab'
+        cal.prodid = 'GoVocal'
```

Comment:
Thought I'd change this whilst I was here

Reply:
Do you know if it's also possible to set this to "Go Vocal" (with a space)? Officially there should be a specify, but if this is indeed more of an identifier, then "GoVocal" is probably the better choice.